src/main/java/com/qaprosoft/carina/demo/gui/pages/AutomationPracticeContactPage.java

```diff
@@ -0,0 +1,51 @@
+package com.qaprosoft.carina.demo.gui.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class AutomationPracticeContactPage extends AbstractPage {
+
+    @FindBy(id = "id_contact")
+    private ExtendedWebElement subjectHeading;
+
+    @FindBy(id = "email")
+    private ExtendedWebElement emailAddress;
```

Comment:
better to name elements with type of element in the end. for example: `emailAddressField`, `emailAddressInputField`, `emailAddressInput`

Reply:
Thank you! I will fix it